maintainers/maintainer-list.nix

```diff
@@ -7922,6 +7922,12 @@
     githubId = 895853;
     name = "Frank Doepper";
   };
+  wohanley = {
```

Comment:
Usually there's a separate commit with:

```
maintainers: add wohanley
```

Reply:
I've seen both versions on PRs. But I agree, I like the separate commit better :)